src/register/RegistrationPage.jsx

```diff
@@ -177,8 +178,16 @@ const RegistrationPage = (props) => {
 
       // This is used by the "User Retention Rate Event" on GTM
       setCookie(getConfig().USER_RETENTION_COOKIE_NAME, true);
+
```

Comment:
Add a note stating the temporary nature of this code

Reply:
I have added the note please see if it makes sense